js/src/modals/WalletSettings/walletSettingsStore.js

```diff
@@ -203,7 +194,9 @@ export default class WalletSettingsStore {
     });
   }
 
-  constructor (api, wallet) {
+  constructor (api, props) {
```

Comment:
`constructor (api, { wallet, onClose })`

(Just so it is obvious what to expect, as opposed to passing anything in)